src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Color {
+    White,
```

Comment:
Enum fields should be in upper case, e.g.
```suggestion
    WHITE,
```